CONTRIBUTING.md

```diff
@@ -85,8 +86,8 @@ The submitted code should adhere to the following:
 
 ### Don'ts
 
-* Any vaguely worded issue will be immediately closed. If there is inactivity on an issue for
-  an extended period of time, it will also be closed.
+* Any vaguely worded issue will be immediately closed.
+* If there is inactivity on an issue for an extended period of time, it will be closed.
```

Comment:
If there is inactivity on an issue for an extended period of time, it will be closed. Please update the issue and re-open it in case you find that it still persists.